internal/providers/terraform/aws/waf_web_acl.go

```diff
@@ -29,7 +29,7 @@ func NewWafWebACL(d *schema.ResourceData, u *schema.UsageData) *schema.Resource
 		region,
 		"Web ACL usage",
 		"months",
-		"USE1-WebACL",
+		".*[^ShieldProtected]-WebACL",
```

Comment:
What is [^ShieldProtected]?

Reply:
The usagetypes have different prices so this regex zooms-in on the correct one. ShieldProtect is DDoS protection across the org resources, and has different [TF resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/shield_protection) as well as [pricing](https://aws.amazon.com/shield/pricing/).
```
	"USW1-WebACL", (shows $5)
	"USW1-ShieldProtected-WebACL", (shows $0)
```

Reply:
Fixes the regex - thanks!